<launch>
    <!-- <arg name="bag_basename" default="20230207_y050_v000_10_pause_025_Hayashide"/> -->
    <arg name="bag_basename" default="20230207_y050_v060_1_Hayashide"/>
    <!-- <arg name="avi_basename" default="20230203_d_060_0_Hayashide"/> -->

    <!-- <arg name="bag_path" default="/media/hayashide/KIOXIA/hayashide/rosbag/0108/yoshinari/$(arg bag_basename).bag"/> -->
    <arg name="bag_path" default="/home/hayashide/catkin_ws/src/ytlab_zed/ytlab_zed_modules/rosbag/$(arg bag_basename).bag"/>
    <arg name="csv_path" default="/home/hayashide/catkin_ws/src/sotsuron_experiment/results/0108/csv/$(arg bag_basename).csv"/>
    <!-- <arg name="topic_name" default="/hsrb/realsense/camera/color/image_raw"/> -->
    <!-- <arg name="fps" default="30"/> -->
    <!-- <arg name="save_path" default="/home/hayashide/catkin_ws/src/sotsuron_experiment/images/results/$(arg avi_basename).avi"/> -->
    <node
        pkg="rosbag" 
        name="play" 
        type="play" 
        args="$(arg bag_path)"
    />    

    <node
        pkg="sotsuron_experiment" 
        name="velocity_plotter" 
        type="velocity_plotter.py" 
        output="screen" 
        args="$(arg bag_basename)"
    />

    <!-- <node
        pkg="image_view" 
        name="video_recorder" 
        type="video_recorder" 
        output="screen" 
        args="image:='$(arg topic_name)' _fps:='$(arg fps)' _filename:='$(arg save_path)'"
    /> -->
</launch>